corehq/apps/hqmedia/templates/hqmedia/references.html

```diff
@@ -68,61 +68,29 @@
-          </a>
-        </h3>
-      </div>
-      <div class="panel-body collapse in"
-           data-bind="attr: { id: id }">
-        <div class="accordion-inner">
-          <p>{% trans 'Showing all media in the module named' %} '<span data-bind="text: name"></span>'.</p>
-          <div class="accordion" data-bind="attr: { id: id + '-ModuleRefs' }">
-            {% include "hqmedia/partials/reference_table.html" with ref_type="active_menu_references" show_refs="showMenuRefs" parent_accordion="ModuleRefs" this_accordion="ModuleMenuRefs" title="Menu References" %}
-          </div>
-          <!-- ko foreach: forms -->
-          <div data-bind="visible: showForm">
-            <h4>{% trans 'Form' %}: <span data-bind="text: name"></span></h4>
-            <div class="accordion" data-bind="attr: { id: id + '-FormRefs' }">
-              {% include "hqmedia/partials/reference_table.html" with ref_type="active_menu_references" show_refs="showMenuRefs" parent_accordion="FormRefs" this_accordion="FormMenuRefs" title="Menu References" %}
-              {% include "hqmedia/partials/reference_table.html" with ref_type="active_images" show_refs="showImageRefs" parent_accordion="FormRefs" this_accordion="FormImageRefs" title="Image References" %}
-              {% include "hqmedia/partials/reference_table.html" with ref_type="active_audio" show_refs="showAudioRefs" parent_accordion="FormRefs" this_accordion="FormAudioRefs" title="Audio References" %}
-              {% include "hqmedia/partials/reference_table.html" with ref_type="active_video" show_refs="showVideoRefs" parent_accordion="FormRefs" this_accordion="FormVideoRefs" title="Video References" %}
-            </div>
-          </div>
-          <!-- /ko -->
-        </div>
-      </div>
-      <!-- /ko -->
-    </article>
+    {% include "hqmedia/partials/reference_table.html" %}
 
     <script type="text/html" id="image-preview-template">
-      <img data-bind="attr: { src: thumb_url }" />&nbsp;&nbsp;
-      <a target="_blank"
-         class="btn btn-default preview-media"
-         data-bind="attr: { href: url }"
-         data-toggle="tooltip" data-title="{% trans 'Opens image in new tab' %}.">{% trans 'View Larger' %}</a>
+      <a target="_blank" data-bind="attr: { href: url }">
+        <img class="preview-media"
+             data-bind="attr: { src: thumb_url }"
+             data-toggle="tooltip"
+             data-title="{% trans 'Open image in new tab' %}." />
+      </a>
     </script>
 
     <script type="text/html" id="audio-preview-template">
       <a target="_blank"
          class="btn btn-default preview-media"
          data-bind="attr: { href: url }"
-         data-toggle="tooltip" data-title="{% trans 'Opens file in new tab' %}.">{% trans 'Hear Audio' %}</a>
+         data-toggle="tooltip" data-title="{% trans 'Open file in new tab' %}.">{% trans 'Hear Audio' %}</a>
```

Comment:
This was pre-existing, but "Hear Audio" and "View Video" sound kind of weird to me (hear more so than view).

"Listen to Audio" and "Play Video" feel like they are more in line with what other sites use. Though I'm not sure if there's some guidelines to follow here

Reply:
That has bugged me, too, updating in 34d09db60a6